web-wallet/src/lib/components/GasControls/GasControls.svelte

```diff
@@ -1,95 +1,91 @@
-<svelte:options immutable={true}/>
+<svelte:options immutable={true} />
 
 <script>
-	import { createEventDispatcher, onMount } from "svelte";
-	import { isType } from "lamb";
-	import { Textbox } from "$lib/dusk/components";
+  import { createEventDispatcher, onMount } from "svelte";
+  import { isType } from "lamb";
+  import { Textbox } from "$lib/dusk/components";
 
-	/** @type {Number} */
-	export let limit;
+  /** @type {Number} */
+  export let limit;
 
-	/** @type {Number} */
-	export let limitLower;
+  /** @type {Number} */
+  export let limitLower;
 
-	/** @type {Number} */
-	export let limitUpper;
+  /** @type {Number} */
+  export let limitUpper;
 
-	/** @type {Number} */
-	export let price;
+  /** @type {Number} */
+  export let price;
 
-	/** @type {Number} */
-	export let priceLower;
+  /** @type {Number} */
+  export let priceLower;
 
-	const dispatch = createEventDispatcher();
+  const dispatch = createEventDispatcher();
 
-	function checkGasLimits () {
-		let isValidPrice = false;
-		let	isValidLimit = false;
-		let isValidGas = false;
+  function checkGasLimits() {
+    let isValidPrice = false;
+    let isValidLimit = false;
+    let isValidGas = false;
 
-		if ([price, limit].every(isType("Number"))) {
-			isValidPrice = price >= priceLower && price <= limit;
-			isValidLimit = limit >= limitLower && limit <= limitUpper;
-			isValidGas = isValidPrice && isValidLimit;
-		}
+    if ([price, limit].every(isType("Number"))) {
+      isValidPrice = price >= priceLower && price <= limit;
+      isValidLimit = limit >= limitLower && limit <= limitUpper;
+      isValidGas = isValidPrice && isValidLimit;
+    }
 
-		dispatch("gasSettings", {
-			isValidGas: isValidGas,
-			limit: limit,
-			price: price
-		});
-	}
+    dispatch("gasSettings", {
+      isValidGas: isValidGas,
+      limit: limit,
+      price: price,
+    });
+  }
 
-	onMount(() => {
-		checkGasLimits();
-	});
+  onMount(() => {
+    checkGasLimits();
+  });
 </script>
 
 <label for={undefined} class="gas-control">
-	<span class="gas-control__label">
-		Price: (lux)
-	</span>
-	<Textbox
-		bind:value={price}
-		className="gas-control__input"
-		max={limit}
-		min={priceLower}
-		on:input={checkGasLimits}
-		required
-		type="number"
-	/>
+  <span class="gas-control__label"> Price: (lux) </span>
```

Comment:
Seems like there's an additional whitespace here.

Reply:
Where exactly? I do not see it.

Reply:
Ok, yes now i see it :)